src/axom/inlet/examples/verification.cpp

```diff
@@ -30,12 +30,13 @@ int main()
 
   // _inlet_workflow_verification_start
   v->registerVerifier([&]() -> bool {
-    int dim;
-    myInlet->get("dimensions", dim);
-    int value;  // field value doesnt matter just that it is present in input file
-    bool x_present = v->hasField("x") && myInlet->get("vector/x", value);
-    bool y_present = v->hasField("y") && myInlet->get("vector/y", value);
-    bool z_present = v->hasField("z") && myInlet->get("vector/z", value);
+    int dim = (*myInlet)["dimensions"];
+    bool x_present = v->hasField("x") &&

```

Comment:
Minor: For a vector with "x", "y" and "z", one might expect the type to be `double` rather than `int`

Reply:
I agree - I think my only changes to this example were to switch to the new accessors but it won't be a problem to change these to doubles.